packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -566,7 +562,8 @@ async function checkSync (transfer) {
     ...transfer,
     completedStep: SYNC,
     nearOnEthClientBlockHeight,
-    status: status.ACTION_NEEDED
+    status: status.ACTION_NEEDED,
+    proof // used when checkSync() is called by unlock()
   }
```

Comment:
Should we put this return inside the `if` statement

Reply:
I liked the ACTION_NEEDED at the end so that it will be easy so handle auto finalization to Ethereum if/when we add it.